starknet_py/serialization/serialization_test.py

```diff
@@ -4,9 +4,9 @@
 import json
 from typing import NamedTuple
 
-from starknet_py.abi.parser import AbiParser
-from starknet_py.abi.v1.parser import AbiParser as AbiParserV1
-from starknet_py.abi.v2.parser import AbiParser as AbiParserV2
+from starknet_py.abi.v0 import AbiParser
```

Comment:
nit: Also in other files
```suggestion
from starknet_py.abi.v0 import AbiParser as AbiParserV0
```